game/scripts/npc/abilities/death_prophet_spirit_siphon.txt

```diff
@@ -47,7 +47,7 @@
       "AbilityChargeRestoreTime" //OAA
       {
         "value"                                           "35" // special_bonus_unique_death_prophet_5_oaa
-        "special_bonus_unique_death_prophet_5"            "-22"
+        "special_bonus_unique_death_prophet_5"            "-15" // special_bonus_unique_death_prophet_5 is not used
```

Comment:
keep in mind this is the worse 25 talent on DP

Reply:
This is not a nerf. I know it's worse. We use special_bonus_unique_death_prophet_5_oaa which changes the cd by 15, I just changed this as a reminder. Since DP gains cdr per hero lvl this talent becomes bonkers at lvl 50.